runtime/compiler/ilgen/Walker.cpp

```diff
@@ -4861,6 +4861,7 @@ TR_J9ByteCodeIlGenerator::runMacro(TR::SymbolReference * symRef)
                   break;
                case 'L':
                case '[':
+               case 'Q':
```

Comment:
My current understanding is that the component type of an array will not be specified with a `Q` for value types, so this shouldn't need to check for that case.